handler_test.go

```diff
@@ -639,16 +636,20 @@ func TestHandler_AuthenticatedDatabases_AuthorizedBasicAuth(t *testing.T) {
 
 // Utility functions for this test suite.
 
-func MustHTTP(verb, url, body string) (int, string) {
-	return MustHTTPWithHeaders(verb, url, nil, body)
-}
-
-func MustHTTPWithHeaders(verb, url string, headers map[string]string, body string) (int, string) {
-	req, err := http.NewRequest(verb, url, bytes.NewBuffer([]byte(body)))
+func MustHTTP(verb, path string, params, headers map[string]string, body string) (int, string) {
+	req, err := http.NewRequest(verb, path, bytes.NewBuffer([]byte(body)))
 	if err != nil {
 		panic(err)
 	}
 
+	if params != nil {
+		q := url.Values{}
+		for k, v := range params {
+			q.Set(k, v)
+		}
+		req.URL.RawQuery = q.Encode()
+	}
+
```

Comment:
I tried to do this, but it didn't work. Am I missing something?

`url.Values{params}`